util/http.go

```diff
@@ -111,15 +111,15 @@ func ExtractAuth(c echo.Context) (string, error) {
 		return "", &HttpError{
 			Code:    http.StatusUnauthorized,
 			Reason:  ERR_INVALID_AUTH,
-			Details: "invalid api key was specified",
+			Details: "Unsupported authorization scheme: Bearer is a required prefix, also make sure you have a valid and unexpired API key. An API key is required to use this endpoint.",
 		}
 	}
 
 	if parts[0] != "Bearer" {
 		return "", &HttpError{
 			Code:    http.StatusUnauthorized,
 			Reason:  ERR_AUTH_MISSING_BEARER,
-			Details: "invalid api key was specified",
+			Details: "Unsupported authorization scheme: Bearer is a required prefix, also make sure you have a valid and unexpired API key. An API key is required to use this endpoint.",
```

Comment:
Can we just put all hardcoded strings in a constant? This so we we can edit or add the message in a single source and call the same constant variable instead of repeatedly hardcoding the same text over.

Reply:
And please merge this against `dev`.